types/screeps/index.d.ts

```diff
@@ -1757,9 +1980,19 @@ interface _Constructor<T> {
 }
 
 interface _ConstructorById<T> extends _Constructor<T> {
-    new (id: string): T;
-    (id: string): T;
+    new (id: Id<T>): T;
+    (id: Id<T>): T;
+}
+
+// tslint:disable-next-line: no-namespace
```

Comment:
Why disable these lint rules?

Reply:
On review they are unnecessary and have been removed in latest commit. Thanks.